src/utils.cpp

```diff
@@ -144,6 +144,18 @@ std::string ToCIFS(std::string& UNCName)
   return CIFSname;
 }
 
+bool insertUser(std::string& UNCName)
+{
+  std::string SMBPrefix = "smb://" + g_szUser + ":" + g_szPass + "@";
+  size_t found;
+  if ((found = UNCName.find("smb://")) != std::string::npos)
```

Comment:
I hate assignments in if clauses... (but this ofc just personal taste.)

Reply:
And, `found` has to be `0`, right? If `smb://`appears in the middle of the string, ... snafu.